[DOCS-694] RUM new dashboards (#6768)
DataDog/documentation: content/en/real_user_monitoring/dashboards/performance_overview_dashboard.md

---
title: RUM Performance overview Dashboard
kind: documentation
further_reading:
    - link: '/real_user_monitoring/installation/advanced_configuration'
      tag: 'Documentation'
      text: 'Advanced configuration for RUM data collection'
    - link: '/real_user_monitoring/explorer'
      tag: 'Documentation'
      text: 'Explore your views within Datadog'
---

> The performance overview dashboard offers a bird’s-eye view of RUM applications. It is separated in 3 sections:

Reviewer: ```suggestion
The performance overview dashboard offers a bird’s-eye view of RUM applications. It is separated in three sections:
```

- **Performance metrics**:

>     For all views, 3 browser metrics are highlighted: First Contentful Paint, DOM Content Loaded, and Load Event. For each one of these metrics, widgets show the median, the 75th percentile, and the 90th percentile.

Reviewer: ```suggestion
    For all views, three browser metrics are highlighted: First Contentful Paint, DOM Content Loaded, and Load Event. For each one of these metrics, widgets show the median, the 75th percentile, and the 90th percentile.
```

- **Trends**:

>     Visualize the evolution of page views, front-end errors related to back-end calls failing, JS errors, and long tasks.

Reviewer: ```suggestion
    Visualize the evolution of page views, frontend errors related to backend calls failing, JS errors, and long tasks.
```
